src/NuGet.Core/NuGet.Packaging/Signing/Utility/CertificateChainUtility.cs

```diff
@@ -184,14 +184,31 @@ internal static bool BuildCertificateChain(X509Chain chain, X509Certificate2 cer
                 throw new ArgumentNullException(nameof(certificate));
             }
 
-            var buildSuccess = chain.Build(certificate);
+            bool buildSuccess = BuildWithPolicy(chain, certificate);
```

Comment:
For my own knowledge, are we getting rid of `var` when we update code?  I prefer to not use it but was wondering what our policy is.

Reply:
@jeffkl - please refer to `rule no. 10` in https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md#c-coding-style